crates/biome_js_analyze/src/lint/nursery/use_trim_start_end.rs

```diff
@@ -0,0 +1,181 @@
+    /// See the MDN documentation for more details:
+    /// - [String.prototype.trimStart()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart)
+    /// - [String.prototype.trimEnd()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd)
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// const foo = bar.trimLeft();
+    /// ```
+    ///
+    /// ```js,expect_diagnostic
+    /// const foo = bar.trimRight();
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```js
+    /// const foo = bar.trimStart();
+    /// ```
+    ///
+    /// ```js
+    /// const foo = bar.trimEnd();
+    /// ```
+    ///
+    pub UseTrimStartEnd {
+        version: "next",
+        name: "useTrimStartEnd",
+        language: "js",
+        recommended: false,
+        sources: &[RuleSource::EslintUnicorn("prefer-string-trim-start-end")],
+        fix_kind: FixKind::Safe,
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct UseTrimStartEndState {
+    member_name: TokenText,
+    span: TextRange,
+    replaced_member_name: String,
+}
+
+impl Rule for UseTrimStartEnd {
+    type Query = Ast<JsCallExpression>;
+    type State = UseTrimStartEndState;
+    type Signals = Option<Self::State>;
+    type Options = ();
+
+    fn run(ctx: &RuleContext<Self>) -> Self::Signals {
+        let node = ctx.query();
+        let arguments = node.arguments().ok()?;
+        let args = arguments.args();
+
+        if !args.is_empty() {
+            // If arguments are present, it suggests this function call may not be intended for `String.trimStart()` or `String.trimEnd()`,
+            // as these methods do not accept parameters according to the specification:
+            // - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart#parameters
+            // - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd#parameters
+            return None;
+        }
+
+        let callee = node.callee().ok()?;
+        let token = generate_syntax_token(callee)?;
+        let suggested_name = suggested_name(&token);
+
+        Some(UseTrimStartEndState {
+            member_name: token.token_text_trimmed(),
+            span: token.text_range(),
+            replaced_member_name: suggested_name,
+        })
+    }
+
+    fn diagnostic(_: &RuleContext<Self>, state: &Self::State) -> Option<RuleDiagnostic> {
+        let member_name = state.member_name.text();
+        let replaced_member_name = state.replaced_member_name.clone();
+
+        let diagnostic_message = markup! {
+            "Use "{replaced_member_name}" instead of "{member_name}"."
+        }
+        .to_owned();
+        let note_message = {
+            markup! {
+                ""{member_name}"() is an alias for "{replaced_member_name}"."
```

Comment:
We should be consistent here:
```suggestion
                ""{member_name}" is an alias for "{replaced_member_name}"."
```